models/user.js

```diff
@@ -394,6 +410,55 @@ UserService.findByIdArray = (ids) => {
 };
 
 /**
+ * Creates a JWT from a user email. Only works for local accounts.
+ * @param {String} email of the local user
+ */
+UserService.createJWT = function (email) {
```

Comment:
Should be named something more specific to the intention, like `UserService.createPasswordResetToken` which describes its use case.